test/vm_tests/builder/codegen_test.dart

```diff
@@ -29,12 +29,12 @@ import './util.dart';
 
 main() {
   group('ImplGenerator', () {
-    ImplGenerator implGenerator;
+    ImplGenerator? implGenerator;
 
-    MockLogger logger;
-    SourceFile sourceFile;
+    late MockLogger logger;
+    late SourceFile sourceFile;
     CompilationUnit unit;
```

Comment:
Doesn't this also need to be `late`? Or no - since its not a runtime thing?

Reply:
It looks like in this case it's because it's only ever referenced after it's definitely assigned within setUpAndParse, so it didn't need to be late.